Source/Atomic/UI/UIEditField.h

```diff
@@ -72,6 +72,10 @@ class UIEditField : public UIWidget
 
     virtual bool OnEvent(const tb::TBWidgetEvent &ev);
 
+    virtual void OnFocusChanged(bool focused);
+
+    // Used to keep track of if we have just been focused for the click select
+    bool firstFocusFlag_;
```

Comment:
This could be a private member?   Also, I don't see that it is being initialized in the constructor so it will have an undefined value